instrumentation/koala/lib/opentelemetry/instrumentation/koala/patches/instrumentation.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Koala
+      module Patches
+        # Module to prepend to Koala::Facebook::API for instrumentation
+        module Instrumentation
+          VERBS_TO_SPAN_NAMES = Hash.new { |h, k| h[k] = "Koala #{k}" }
+
+          def graph_call(path, args = {}, verb = 'get', options = {}, &post_processing)
```

Comment:
@fbogsany @robertlaurin Is there a better way to take in any arguments, take out what I need for tracing and then pass it unaltered to koala?

Reply:
Is it even a good idea to do something like this?

Reply:
I don't think so - this seems correct to me.